src/Components/test/testassets/BasicTestApp/InteropComponent.razor

```diff
@@ -175,6 +175,22 @@
             ReturnValues["roundTripByteArrayWrapperObjectFromDotNet"] = ((IJSInProcessRuntime)JSRuntime).Invoke<ByteArrayInterop.ByteArrayWrapper>("roundTripByteArrayWrapperObject", byteArrayWrapperObject).ToString();
         }
 
+        // Only run this test for Blazor Server
+        if (shouldSupportSyncInterop)
```

Comment:
Shouldn't this be !shouldSupportSyncInterop since the comment references Blazor server

Reply:
https://github.com/dotnet/aspnetcore/pull/33491/files#r653772346